test/functional/feature_rbf.py

```diff
@@ -627,6 +630,15 @@ def test_no_inherited_signaling(self):
         assert_equal(True, self.nodes[0].getmempoolentry(optin_parent_tx['txid'])['bip125-replaceable'])
         assert_raises_rpc_error(-26, 'txn-mempool-conflict', self.nodes[0].sendrawtransaction, replacement_child_tx["hex"], 0)
 
+    def test_replacement_relay_fee(self):
+        wallet = MiniWallet(self.nodes[0])
+        wallet.scan_blocks(start=77, num=1)
+        tx = wallet.send_self_transfer(from_node=self.nodes[0])['tx']
+
+        # Higher fee, higher feerate, different txid, but the replacement does not provide a relay
+        # fee conforming to node's `incrementalrelayfee` policy of 1000 sat per KB.
+        tx.vout[0].nValue -= 1
```

Comment:
(nit) was slightly surprised that 1 was parsed as 1 satoshi and that the tx didn't need a new signature

```suggestion
        # Increase the fee by 1 satoshi. Same signature works because it is an anyone-can-spend.
        tx.vout[0].nValue -= Decimal("0.00000001")
```

Reply:
The test wouldn't pass with the suggestion applied